Makefile

```diff
@@ -17,7 +17,8 @@ all: build
 
 SOURCEDIR=./ecr-login
 SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
-BINARY_NAME=docker-credential-ecr-login
+VERSION := $(shell cat VERSION.txt)
+BINARY_NAME=docker-credential-ecr-login-$(VERSION)
```

Comment:
Can you revert this change to the `BINARY_NAME`?  The resulting binary name is still `docker-credential-ecr-login` in `build_binary.sh` (see line 37 of that file) and changing it here breaks `make`'s ability to track unnecessary rebuilds.  (With this change, `make linux-amd64` rebuilds every time you invoke it, but before this change `make` will tell you "make: Nothing to be done for 'linux-amd64'." when invoked multiple times.)


Also: Docker depends on the exact binary name and I'd like to keep it easy for people to just copy the binary into their `PATH`; when you specify `"credsStore": "X"`, Docker looks for a binary named `docker-credential-X`.

Reply:
Fixed by 9246903